ui/app/AppLayouts/Chat/controls/community/AddressesInputList.qml

```diff
@@ -0,0 +1,169 @@
+                    when: !model.valid
+
+                    PropertyChanges {
+                        target: delegate
+
+                        color: Theme.palette.alphaColor(
+                                   Theme.palette.dangerColor1, 0.05)
+                    }
+
+                    PropertyChanges {
+                        target: statusIcon
+
+                        width: 21
+                        height: 21
+                        icon: "warning"
+                        color: Theme.palette.dangerColor1
+                    }
+
+                    PropertyChanges {
+                        target: addressText
+
+                        color: Theme.palette.dangerColor1
+                    }
+                }
+
+                StatusIcon {
+                    id: statusIcon
+
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.horizontalCenter: parent.left
+                    anchors.horizontalCenterOffset: 18
+
+                    width: 16
+                    height: 16
+                    icon: "checkbox"
+                    color: Theme.palette.successColor1
+                }
+
+                StatusBaseText {
+                    id: addressText
+
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.left: parent.left
+                    anchors.right: deleteIcon.left
+                    anchors.margins: 7
+                    anchors.leftMargin: 34
+
+                    color: Theme.palette.directColor1
+
+                    font.pixelSize: 15
+                    font.weight: Font.Medium
+
+                    elide: Text.ElideMiddle
+                    text: model.address
+                }
+
+                StatusIcon {
+                    id: deleteIcon
+
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.right: parent.right
+                    anchors.rightMargin: 10
+
+                    width: 16
+                    height: 16
+
+                    icon: "delete"
+                    color: Theme.palette.directColor1
+
+                    MouseArea {
+                        anchors.fill: parent
+                        cursorShape: Qt.PointingHandCursor
+
+                        onClicked: {
+                            root.removeAddressRequested(model.index)
+                        }
+                    }
+                }
+            }
+        }
+
+        StatusBaseInput {
+            id: input
+
+            showBackground: false
+            maximumLength: 2000
+
+            width: root.availableWidth
+
+            leftPadding: 0
+            rightPadding: 0
+
+            multiline: true
+
+            height: Math.min(root.maximumTextInputHeight,
+                             edit.implicitHeight + topPadding + bottomPadding)
+
+            placeholderText: qsTr("Example: 0x39cf...fbd2")
+
+            Keys.onReturnPressed: {
```

Comment:
What about `onEnterPressed`? ;)

Reply:
`onEnterPressed` seems to be problematic here bc of `StatusBaseInput` internal details. `onReturnPressed` just works as expected...

Reply:
Yeah, for the regular "Return" key sure, have you tried typing the number on a numpad and pressing the numpad "Enter"? ;)

Reply:
Fixed! Thanks for pointing that :)